src/03-control-flow.js

```diff
@@ -86,7 +106,24 @@ function positiveNegativeOrZero(a) {}
  *  aroundTheWorldGreeting();
  *  //> "Hello World"
  */
-function aroundTheWorldGreeting(language) {}
+function aroundTheWorldGreeting(language) {
+  const engMsg = "Hello World";
+  const spanMsg = "Hola Mundo";
+  const gerMsg = "Hallo Welt";
+  if (language === 'spanish') {
+    return spanMsg;
+  }
+  else if (language === 'german') {
+    return gerMsg;
+  }
+  else if (language === 'english') {
+    return engMsg;
+  } else if (language !== 'spanish' || language !== 'german' || language !== 'english') {
```

Comment:
This is a good place to just use an else. Since you have conditionals that are already accounting for spanish, german, and english, anything else will false under the else.